.gitignore

```diff
@@ -34,6 +34,7 @@ Thumbs.db
 /node_modules
 /bower_components
 npm-debug.log
+yarn.lock
```

Comment:
The `yarn.lock` is set in .gitattributes as a binary file and shouldn't be ignored. See notes in #1226